servicetalk-buffer-api/src/main/java/io/servicetalk/buffer/api/BufferHolder.java

```diff
@@ -17,7 +17,11 @@
 
 /**
  * An object which contains a {@link Buffer}.
+ *
+ * @deprecated This API is going to be removed in the next minor version with no planned replacement. If it cannot be
```

Comment:
Bcz we are not quite correct in semantic versioning until 1.0.0, consider saying "in the future releases" instead of giving more specifics on minor/major version